src/core/reactor.ts

```diff
@@ -3192,6 +3184,49 @@ export class FederatedApp extends App {
         return this.greatestTimeAdvanceGrant;
     }
 
+    /**
+     * Return whether the next event can be handled, or handling the next event
+     * has to be postponed to a later time.
+     * 
+     * If this federated app has not received a sufficiently large time advance
+     * grant (TAG) from the RTI for the next event, send it a Next Event Time
+     * (NET) message and return. _next() will be called when a new greatest TAG
+     * is received. The NET message is not sent if the connection to the RTI is
+     * closed. FIXME: what happens in that case? Will next be called?
+     * @param event 
+     */
+    protected canProceed(event: TaggedEvent<Present>) {
+        if (this._isRTISynchronized() && this._isActive()) {
+            // FIXME: Why would it proceed if the reaction is inactive?
```

Comment:
It looks like the check for `this._isActive()` was a bandaid I added here when I was hunting down a bigger bug and isn't necessary. I'll remove it.

Reply:
Shouldn't `canProceed` return immediately if the reactor is not active, without contacting the RTI?

Reply:
In your suggestion, how should it return? True or false? My understanding of the purpose of TAGs in the RTI is that logical time in a federate should only be allowed to advance up to the greatest received TAG. Why should an inactive FederatedApp get special treatment?

Reply:
You're right. This is not the right place to do this check anyway. You can removed it.